CONTRIBUTING.md

```diff
@@ -8,11 +8,39 @@ Community contributions rock and we are psyched you're reading this document!
 - Feature requests are welcome and go [here](https://github.com/stjude-rust-labs/wdl/discussions/categories/feature-requests)
 - Lint rule proposals go [here](https://github.com/stjude-rust-labs/wdl/discussions/categories/rule-proposals)
 
+## How can I start contributing?
+
+### I don't want to write code, can I still contribute?
+
+Sure!
+
+You can always open a [discussion](https://github.com/stjude-rust-labs/wdl/discussions/categories/rule-proposals) with a proposal for a new lint rule or other enhancement as well contributing to any open discussions. Ensure that you provide a clear and concise title and description of the desired addition. For lint rules in particular, WDL examples and/or links to the relevant portion of the WDL spec are extremely useful.
+
+We also welcome bug reports. If you discover a flaw in our codebase, please review the list of open issues to ensure that it is not a duplicate. Please also attempt to debug the issue locally and ensure that it is not a configuration issue. Once you have done both, please file a new issue providing the relevant information in the issue. Please provide the exact steps to reproduce the problem, specific example(s) that demonstrate the steps, and the behavior you observe as well as the behavior you expected to observe. For terminal-based use cases, a copy and paste of the command and error log (please use markdown formatting appropriately) is preferred. For interactive use cases (such as the VSCode extension), screenshots and/or GIFs are welcome ways to provide additional information to maintainers.
+
+We also appreciate feedback on our documentation. Feel free to look over any of our `*.md` files and note any issues you find. You can also explore our lint rule documentation by [installing `sprocket`](https://stjude-rust-labs.github.io/sprocket/installation.html) and reading the output of `sprocket explain`. (n.b.: we hope to replace `sprocket explain` with a website where each rule will have a dedicated page, but that has not been realized yet).
+
+The maintainers reserve the right to close issues and discussions as deemed necessary as well as to delete comments and interactions within the repository.
```

Comment:
I like this sentence, but it should be moved somewhere else. It doesn't make sense under this header

Reply:
I put it here because the preceding paragraphs describe opening issues and discussions.

Reply:
Good point. That made me step back, and I think that the bug report paragraph might also belong on a different header than "I don't want to write code...". The bug report instructions are just applicable to anyone reading this document, and so is this sentence I've highlighted.

Reply:
To me, these are all of the non-code contribution methods: discussions, bug reports, and documentation feedback. The general instructions for a bug report are also in the bug report template in #415. I think those serve distinct purposes. This document is to guide people on the methods for contributing. The template is to facilitate helpful error reporting. I think plenty of people would jump straight to filing an issue if they find a problem without reading the CONTRIBUTING guide. However for new people coming to the project and wondering how to contribute, this outlines the various ways to get involved.

Reply:
Rust has a very concise contributing guide that mostly links to other documents: https://github.com/rust-lang/rust/blob/master/CONTRIBUTING.md. Rails takes a more verbose approach to outlining different contribution types: https://github.com/rails/rails/blob/main/CONTRIBUTING.md.

Reply:
Python's isn't on GitHub, but has a lot of information: https://devguide.python.org/#contributing